src/append/rolling_file/policy/compound/trigger/time.rs

```diff
@@ -73,6 +72,20 @@ impl Default for TimeTriggerInterval {
     }
 }
 
+#[derive(Debug, Error)]
+enum TimeTrigerIntervalError {
+    #[error("The 'Seconds' time trigger interval value is out of bounds, ensure it fits within an i64: : '{0:?}'")]
+    Second(TimeTriggerInterval),
+    #[error("The 'Minutes' time trigger interval value is out of bounds, ensure it fits within an i64: : '{0:?}'")]
+    Minute(TimeTriggerInterval),
+    #[error("The 'Hours' time trigger interval value is out of bounds, ensure it fits within an i64: : '{0:?}'")]
+    Hour(TimeTriggerInterval),
+    #[error("The 'Days' time trigger interval value is out of bounds, ensure it fits within an i64: : '{0:?}'")]
+    Day(TimeTriggerInterval),
+    #[error("The 'Weeks' time trigger interval value is out of bounds, ensure it fits within an i64: : '{0:?}'")]
+    Week(TimeTriggerInterval),
+}
+
```

Comment:
Besides, I remember now why I put a specific number here before. The max value of these is not i64::MAX. They should be within i64::MAX after they convert to milliseconds.

Reply:
Action worthy?